diff --git a/scrapers/wi/bills.py b/scrapers/wi/bills.py
index 3df22d5dd..29249fafd 100644
--- a/scrapers/wi/bills.py
+++ b/scrapers/wi/bills.py
@@ -409,6 +409,7 @@ class WIBillScraper(Scraper):
 
         yes_names_count = 0
         no_names_count = 0
+        nv_names_count = 0
 
         for td in doc.xpath("//tbody/tr/td[4]"):
             name = td.text_content()
@@ -419,10 +420,13 @@ class WIBillScraper(Scraper):
                 elif vote_td.text_content() == "N":
                     vote.vote("no", name)
                     no_names_count += 1
-                elif vote_td.text_content() == "NV":
+                elif vote_td.text_content() in ("NV", "x", "X"):
                     vote.vote("not voting", name)
+                    nv_names_count += 1
 
         if yes_names_count != int(vote_counts[0][0]):
             raise ValueError("Yes votes and number of Names doesn't match")
         if no_names_count != int(vote_counts[0][1]):
             raise ValueError("No votes and number of Names doesn't match")
+        if nv_names_count != int(vote_counts[0][2]):
+            raise ValueError("Not-voting votes and number of Names doesn't match")
